apps/web/src/pages/about.tsx

```diff
@@ -9,6 +9,286 @@ export default function Landing() {
         {/* <Button variant="contained" color="primary">
```

Comment:
I recommend you create a component for the header. It will be used a lot and redefining it every time would get tiring fast. Just something to consider.